process/block/shardblock.go

```diff
@@ -24,16 +24,15 @@ const maxCleanTime = time.Second
 // shardProcessor implements shardProcessor interface and actually it tries to execute block
 type shardProcessor struct {
```

Comment:
Can you please reformat the code in this struct also?

Reply:
Done.